packit_service/worker/handlers/testing_farm.py

```diff
@@ -170,7 +213,17 @@ def run_with_copr_builds(self, targets: List[str], failed: Dict):
             event_data["build_targets_override"] = list(targets_without_builds)
             self.run_copr_build_handler(event_data, len(targets_without_builds))
 
-        for target, copr_build in targets_with_builds.items():
+        if not targets_with_builds:
```

Comment:
TODO: if there are copr builds, the handler will trigger for each copr build end, resulting in multiple groups. We should group these together